tests/test_cli.py

```diff
@@ -114,7 +114,21 @@ def test_config_project(self):
                             remove.assert_called_once_with('./nio.conf')
                             self.assertEqual(rename.call_count, 1)
 
-    def test_config_with_no_nioenv(self):
+    def test_config_project_secure(self):
+        pk_host = '123.pubkeeper.nio.works'
+        pk_token = '123123'
+        ws_host = '123.websocket.nio.works'
+        with patch('builtins.open', mock_open()) as mopen:
```

Comment:
In general it's better (mainly for indentation) to comma separate these `with` statements rather than nest them.

So instead of this:
```python
with a as A:
    with b as B:
        # do stuff
```
do
```python
with a as A, b as B:
    # do stuff
```